### PR TITLE
Add ipv6

### PR DESCRIPTION
**Pull request by QuentinPerez**

### CODE REVIEW

File: Linux/tree-common/init [hunk 1]
```diff
@@ -88,6 +88,21 @@ if [ "$(get_opt nameserver)" != "" ]; then
...
 log_end_msg
 
+# Configure ipv6
+if [ "$(get_any NO_IPV6)" != "1" ]; then
+	IPV6=$(scw-metadata --cached | grep "^IPV6=" | cut -d'=' -f2)
+
+	if [ "$IPV6" != "" ]; then
+	    log_begin_msg "Configuring ipv6"
+
+		IP=$(scw-metadata --cached | grep "^IPV6_ADDRESS=" | cut -d'=' -f2)
```
Comment (moul) [inline Linux/tree-common/init]: ==

`IP=$(scw-metadata --cached IPV6_ADDRESS)`

**Comment by QuentinPerez** [comment]
Done

File: Linux/tree-common/init [hunk 2]
```diff
@@ -88,6 +88,21 @@ if [ "$(get_opt nameserver)" != "" ]; then
...
 log_end_msg
 
+# Configure ipv6
+if [ "$(get_any NO_IPV6)" != "1" ]; then
+	IPV6=$(scw-metadata --cached IPV6 | cut -d'=' -f2)
+
+	if [ "$IPV6" != "" ]; then
+	    log_begin_msg "Configuring ipv6"
+
+		IP=$(scw-metadata --cached IPV6_ADDRESS | cut -d'=' -f2)
```
Comment (moul) [inline Linux/tree-common/init]: you can also remove the cut -d=

**Comment by QuentinPerez** [comment]
Arfff, tiredness

**Comment by moul** [comment]
:)

**Comment by moul** [comment]
LGTM

**Pull request merged**